starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -3,13 +3,17 @@
 import com.starter.fullstack.api.Inventory;
 import com.starter.fullstack.dao.InventoryDAO;
 import java.util.List;
+import java.util.Optional;
+
```

Comment:
looks like you still have this empty line. Please remove it. There shouldn't be any empty lines in your imports. Can you also remove the empty line you have in the imports in InventoryDAO.java